src/components/Book.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import {PropTypes} from 'prop-types';
+
+export default function Book(props) {
+    return (
+        <div>
+            <p>{props.book.title} - {props.book.category}</p>
```

Comment:
Hi, you should be using object de structuring.

Reply:
Done